build.py

```diff
@@ -14,32 +14,35 @@
 
 FRAMEWORK_REPO = 'GeositeFramework'
 DEFAULT_ORG = 'CoastalResilienceNetwork'
-DEFAULT_BRANCH = 'master'
+DEFAULT_FRAMEWORK_BRANCH = 'develop'
```

Comment:
Is this a good option for the default?  I suppose the tool is run more often to test a build than to deploy to production, but curious about the reasoning.

Reply:
I'm actually not sure what gets used more often, but I set this to `develop` so that it would be in sync with Github (when we make that change). I'm open to switching it back.